src/art_net/4C_art_net_impl_stationary.hpp

```diff
@@ -87,14 +88,20 @@ namespace Arteries
       FOUR_C_THROW("LoadState() not available for stationary formulation");
     }
 
+    //! collect runtime output data
+    void collect_runtime_output_data();
+
+    //! write data required for restart
+    virtual void output_restart();
```

Comment:
It does not need to be virtual as you do not override it.
```suggestion
    void output_restart();
```